bam_to_mate_hist.py

```diff
@@ -133,17 +136,22 @@ def parse_bam_file(bamfile, num_reads, count_diff_refname_stub=False):
             else:
                 read1_pos = read.reference_start
                 read2_pos = read.next_reference_start
-                dist = str(abs(read1_pos - read2_pos))
-                dists[num] = dist
+                dist = abs(read1_pos - read2_pos)
```

Comment:
could avoid this by using a defaultdict. not a big deal functionally, just somewhat easier to read.